src/Microsoft.IdentityModel.Tokens/LogMessages.cs

```diff
@@ -86,6 +86,10 @@ internal static class LogMessages
         public const string IDX10252 = "IDX10252: RequireSignedTokens property on ValidationParameters is set to false and the issuer signing key is null. Exiting without validating the issuer signing key.";
         public const string IDX10253 = "IDX10253: RequireSignedTokens property on ValidationParameters is set to true, but the issuer signing key is null.";
         public const string IDX10254 = "IDX10254: '{0}.{1}' failed. The virtual method '{2}.{3}' returned null. If this method was overridden, ensure a valid '{4}' is returned.";
+        public const string IDX10255 = "IDX10255: ValidTypes property on ValidationParameters null. Exiting without validating the token type.";
```

Comment:
might want to write: ValidationParameters.ValidTypes is null or empty. Exiting ....

reason the logic checks for null and count > 0

Reply:
Good catch! Fixed this.